Adyen.Test/CloudApiPosRequestTest.cs

```diff
@@ -57,7 +57,7 @@ public void TestCloudApiTransactionStatusResponseSuccess()
 
             try
             {
-                var transactionStatusResponse = (TransactionStatusResponse)saleToPoiResponse.Item;
+                var transactionStatusResponse = (TransactionStatusResponse)saleToPoiResponse.MessagePayload;
```

Comment:
do we do the same for our java lib ?

Reply:
In the dotnet the Item is an object which is a placeholder for the any of the response. We rename it to MessagePayload as it is more clear. In java there are all the responses as members of the SaleToPOIResponse class.